code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const ThoughtForm = ({ newThought, setNewThought, onFormSubmit }) => {
+
+
+    return (
+        <form onSubmit={onFormSubmit}>
+            <h1>What's making you happy right now? </h1>
+            <textarea
+                className="text-input"
+                value={newThought}
+                onChange={(event) => setNewThought(event.target.value)}
+                type="text"
+                id="newThought"
```

Comment:
Here you could add an htmlFor, matching your id (newThought) in order to create validations, such as required as an example